### PR TITLE
Add spacemacs ABC

### PR DESCRIPTION
**Pull request by mudstart**
Thank you for contributing to Spacemacs!

Before you submit this pull request, please ensure it is against the develop branch and not master.

This message should be replaced with a description of your change.

Thank you <3

### CODE REVIEW

**Comment by sdwolfz** [comment]
This sounds like a good idea 👍 but please change the target branch of this PR to `develop`.

File: README.md [hunk 1]
```diff
@@ -107,6 +107,8 @@ Comprehensive documentation is available for each layer by pressing
 You can also check the [general documentation][DOCUMENTATION.org],
 [quick start guide][QUICK_START.org] and the [FAQ][FAQ.org].
 
+Also you can check this amazing playlist create by [Eivind Fonn](https://github.com/TheBB) Spacemacs ABC  [Youtube](https://www.youtube.com/watch?v=ZFV5EqpZ6_s&list=PLrJ2YN5y27KLhd3yNs2dR8_inqtEiEweE)
```
Comment (d12frosted) [inline README.md]: I know that our glorious @emacspace will cover this, but let's avoid unnecessary double spaces: 

```
Spacemacs ABC  [Youtube]
             ^^
```

Comment (mudstart) [reply README.md]: Thanks @d12frosted  for the observations

**Comment by duianto** [comment]
Request for changes to this PR:
- The two commits are on top of the `master` branch, they need to be on top of the `develop` branch.
- Squash both commits into one.
- Add your name to the `Various documentation improvements` section of the `changelog.develop`:
https://github.com/syl20bnr/spacemacs/blob/develop/CHANGELOG.develop#various-improvements

**Pull request closed without merging**